src/solana/SolanaEngine.ts

```diff
@@ -121,13 +124,41 @@ export class SolanaEngine extends CurrencyEngine<
     }
 
     const funcs = this.networkInfo.rpcNodes.map(serverUrl => async () => {
+      const apiKeys = asSolanaInitOptions(this.initOptions) as {
+        [key: string]: string
+      }
+      const regex = /{{(.*?)}}/g
```

Comment:
I don't see why the `?` is needed. This should do the exact same thing: `/{{(.*)}}/g`